code/modules/reagents/chemistry/reagents/reaction_agents_reagents.dm

```diff
@@ -0,0 +1,113 @@
+/datum/reagent/reaction_agent
+    name = "reaction agent"
+    description = "Hello! I am a bugged reagent. Please report me for my crimes. Thank you!!"
+
+/datum/reagent/reaction_agent/intercept_reagents_transfer(datum/reagents/target, amount)
+    if(!target)
+        return FALSE
+    if(target.flags & NO_REACT)
+        return FALSE
+    if(target.has_reagent(/datum/reagent/stabilizing_agent))
+        return FALSE
+    if(LAZYLEN(target.reagent_list) == 0)
+        return FALSE
+    if(LAZYLEN(target.reagent_list) == 1)
+        if(target.has_reagent(type)) //Allow dispensing into self
+            return FALSE
+
```

Comment:
Is this supposed to return `null` if not `FALSE`, or `TRUE`?

Reply:
```suggestion
/datum/reagent/reaction_agent/intercept_reagents_transfer(datum/reagents/target, amount)
    if(!target)
        return FALSE
    if(target.flags & NO_REACT)
        return FALSE
    if(target.has_reagent(/datum/reagent/stabilizing_agent))
        return FALSE
    if(LAZYLEN(target.reagent_list) == 0)
        return FALSE
    if(LAZYLEN(target.reagent_list) == 1)
        if(target.has_reagent(type)) //Allow dispensing into self
            return FALSE
   return TRUE

```

Reply:
Are you using spaces for indentation instead of tabs?

Reply:
I was, this is fixed now